testing/checkin/runner/ops.rs

```diff
@@ -69,37 +70,76 @@ pub fn op_stats_delete(
   test_data.take::<RuntimeActivityStats>(name);
 }
 
-pub struct Stateful {
-  name: String,
+pub struct DOMPoint {
+  pub x: f64,
+  pub y: f64,
+  pub z: f64,
+  pub w: f64,
 }
 
-impl GarbageCollected for Stateful {}
+impl GarbageCollected for DOMPoint {}
 
-impl Stateful {
-  #[op2(method(Stateful))]
-  #[string]
-  fn get_name(&self) -> String {
-    self.name.clone()
+#[op2]
+impl DOMPoint {
+  #[constructor]
+  #[cppgc]
+  fn new(
+    x: Option<f64>,
+    y: Option<f64>,
+    z: Option<f64>,
+    w: Option<f64>,
+  ) -> DOMPoint {
+    DOMPoint {
+      x: x.unwrap_or(0.0),
+      y: y.unwrap_or(0.0),
+      z: z.unwrap_or(0.0),
+      w: w.unwrap_or(0.0),
+    }
   }
 
-  #[op2(fast, method(Stateful))]
-  #[smi]
-  fn len(&self) -> u32 {
-    self.name.len() as u32
+  #[static_method]
+  #[cppgc]
+  fn from_point(
```

Comment:
should we have a `#[rename("fromPoint")]` attribute?

Reply:
It would be great if well-known symbols could also be supported